### PR TITLE
Adding filters and specifying descriptor contents

### PR DESCRIPTION
**Pull request by sudo-bmitch**
This should include changes from #9.

Signed-off-by: Brandon Mitchell <git@bmitch.net>

### CODE REVIEW

**Comment by sudo-bmitch** [comment]
@michaelb990 PTAL

**Comment by sudo-bmitch** [comment]
I also switched to filtering example to use the generic sbom descriptor to align with the previous example output. And I also changed the annotation from `reference` to `referrers`.

**Comment by jdolitsky** [comment]
i think parts of this and #9 are duplicated .. will let the two of you sort this out @sudo-bmitch @michaelb990

**Review by jdolitsky — APPROVED** [review]

**Comment by sudo-bmitch** [comment]
@AkihiroSuda I think this addresses some of your comments in https://github.com/opencontainers/distribution-spec/pull/335. There's probably more we should be doing, but I don't know how much that belongs in blog posts and presentations vs the spec itself.

File: spec.md [hunk 1]
```diff
@@ -496,13 +496,17 @@ When using the `last` query parameter, the `n` parameter is OPTIONAL.
...
 
-To fetch the list of referrers, perform a `GET` request to a path in the following format: `/v2/<name>/referrers/<reference>` <sup>[end-12a](#endpoints)</sup>
+To fetch the list of referrers, perform a `GET` request to a path in the following format: `/v2/<name>/referrers/<reference>` <sup>[end-12a](#endpoints)</sup>.
 
 `<name>` is the namespace of the repository.
 Assuming a repository is found, this request MUST return a `200 OK` response code.
-If a query results in no referrers found, an empty manifest list MUST be returned.
 
-Upon success, the response MUST be a json body in the following format (an index / manifest list):
+Upon success, the response MUST be a json body with an Index containing a list of descriptors.
```
Comment (michaelb990) [inline spec.md]: nit:
```suggestion
Upon success, the response MUST be a JSON body with an Index containing a list of descriptors.
```

File: spec.md [hunk 2]
```diff
@@ -496,13 +496,17 @@ When using the `last` query parameter, the `n` parameter is OPTIONAL.
...
+Each descriptor is of an Image or Artifact manifest in the same `<name>` namespace with a `refers` field that specifies the value of `<reference>`.
+The descriptors MUST include an `artifactType` field that is set to the value of `artifactType` for an Artifact manifest, or the configuration descriptor's `mediaType` for an Image manifest.
```
Comment (michaelb990) [inline spec.md]: I think it is optional to include the `artifactType` in the artifact manifest now, so maybe this should say:
```suggestion
The descriptors MUST include an `artifactType` field that is set to the value of `artifactType` for an Artifact manifest if present, or the configuration descriptor's `mediaType` for an Image manifest.
```

**Review by michaelb990 — CHANGES_REQUESTED** [review]

File: spec.md (hunk 1 above)
Comment (sudo-bmitch) [reply spec.md]: Done, thanks!

File: spec.md (hunk 2 above)
Comment (sudo-bmitch) [reply spec.md]: Done, thanks!

**Review by michaelb990 — APPROVED** [review]

**Pull request merged**